### ISSUE TITLE
[Accessibility] BDD: Redo Part of A11y Self Check After BDD/526 Refactor

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#11895]
## User Story or Problem Statement
We need to perform an accessibility check for BDD after we have combined the two forms.  Since we already did this, this is to really to a quick pass.  Should be able use the same test scripts for the e2e, etc.

### Required:
- [x] Read the [accessibility staging review documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) 
- [x] Conduct axe browser scan, and document or address issues found 
- [x] Ensure end-to-end tests - including the axe check, pass 
- [x] Check color pairing for proper contrast 
- [x] Check for colorblindness issues 
- [x] Zoom to 200% at 1280px wide and confirm readability, no info cut off, etc 
- [x] Repeat for 300% 
- [x] Repeat for 400% 
- [x] Navigate all pages and core functionality with keyboard only 
- [x] Test all pages and core functionality happy path with MacOs + VoiceOver and/or NVDA+Firefox 
    
### Recommended:
- [x] Read the [accessibility post-launch processes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/post-launch-audit-processes.md) 
- [ ] Add keyboard end-to-end tests in CI/CD test runs  
- [x] Conduct axe-coconut browser scan, document or address issues found 
- [x] Test with [WAVE tool](https://wave.webaim.org/) 

## Goal
Get ready for full release of BDD.

## Acceptance Criteria
- [ ] Above checklist is completed

## How to configure this issue
- [x] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)
